saleor/channel/models.py

```diff
@@ -48,6 +48,9 @@ class Channel(ModelWithMetadata):
 
     use_legacy_error_flow_for_checkout = models.BooleanField(default=True)
     automatically_complete_fully_paid_checkouts = models.BooleanField(default=False)
+    draft_order_line_price_freeze_period = models.IntegerField(
```

Comment:
Maybe we should add a comment that the value is in hours?

Reply:
You should also use `PositiveIntegerField` instead of `IntegerField`

Reply:
corrected